package.json

```diff
@@ -32,11 +32,12 @@
     "knex": "^0.13.0",
     "lodash": "^4.17.4",
     "mssql": "^3.3.0",
+    "node-yaml": "^3.1.0",
     "require-sql": "1.0.0",
     "restify": "^4.3.1",
     "sleep-promise": "^2.0.0",
-    "swagger-restify-mw": "^0.7.0",
-    "tedious": "2.0.0"
+    "tedious": "2.0.0",
```

Comment:
I don't think tedious actually needs to be in here - it's only used inside mssql (although that's not related to this change).